src/infrastructure/http.js

```diff
@@ -112,6 +113,10 @@ export default class http {
 	  return EPOCH_ADJUSTMENT;
   }
 
+  static get accountsLabel () {
+  	return  accountsLabel || {};
```

Comment:
mixing tabs and spaces, also double spaces

Reply:
it seem my lint not working well 🤔